lms/djangoapps/mobile_api/mobile_platform.py

```diff
@@ -0,0 +1,59 @@
+"""
+Platform related Operations for Mobile APP
+"""
+import abc
+import re
+
+
+class MobilePlatform:
+    """
+    MobilePlatform class creates an instance of platform based on user agent and supports platform
+    related operations
+    """
+    __metaclass__ = abc.ABCMeta
```

Comment:
The intend is to make MobilePlatform an abstract class and create instance of any of its sub-classes (IOS and Android) with its `get_instance` classmethod